charts/queryservice-gateway/values.yaml

```diff
@@ -4,6 +4,8 @@ platform:
   responseCacheTTL: 3600
   apiBackendHost: api-backend
 
+proxyTimeout: 300
+
 image:
   repository: ghcr.io/wbstack/queryservice-gateway
   tag: "2.2"
```

Comment:
I think we debate this endlessly but this *might* want to either be removed or bumped

Reply:
I'd vote for removing, but it's probably best done across all charts at once?